dist/fluxo.js

```diff
@@ -358,6 +361,16 @@ Fluxo.CollectionStore = Fluxo.Base.extend(
 
     if (storeID) {
       foundStore = this.findWhere({ id: storeID });
+
+      if (!foundStore) {
+        this.stores.some(function(store) {
+          if (store.cid === storeID) {
+            foundStore = store;
+
+            return true;
+          }
+        });
+      }
```

Comment:
Errata: There is an `Array#find` on ES6, although it's not supported by Chrome yet (Safari already supports it :trollface:)

Reference: https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Array/find